src/test/rgw/test_rgw_compression.cc

```diff
@@ -48,3 +115,87 @@ TEST(Decompress, FixupRangePartial)
   ASSERT_EQ(range_t(12, 24), fixup_range(&decompress, 16, 999));
   ASSERT_EQ(range_t(18, 24), fixup_range(&decompress, 998, 999));
 }
+
+TEST(Compress, LimitedChunkSize)
+{
+  CompressorRef plugin;
+  plugin = Compressor::create(g_ceph_context, Compressor::COMP_ALG_ZLIB);
+  ASSERT_NE(plugin.get(), nullptr);
```

Comment:
i think this depends on the test being able to locate a valid ceph.conf. without one, this assertion fails

[test_async_compressor.cc](https://github.com/ceph/ceph/blob/master/src/test/common/test_async_compressor.cc), for example, has to set `plugin_dir` explicitly in order to run as part of `make check`. could you try that here?